pkg/server/operands_handler.go

```diff
@@ -0,0 +1,102 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"os/user"
+	"path/filepath"
+
+	"github.com/openshift/console/pkg/serverutils"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetConfig() (*rest.Config, error) {
+	// If an env variable is specified with the config location, use that
+	if len(os.Getenv("KUBECONFIG")) > 0 {
+		return clientcmd.BuildConfigFromFlags("", os.Getenv("KUBECONFIG"))
+	}
+	// If no explicit location, try the in-cluster config
+	if c, err := rest.InClusterConfig(); err == nil {
```

Comment:
For multi-cluster, we're not going to have a kubeconfig file, and `InClusterConfig` won't work except for the hub cluster. Is there another way to initialize the client with only the API server URL and CA file? It would be good to determine that before merging or it will cause us problems down the road.

Reply:
OK, do you know if we can pass a CA certificate as well?